modules/engage-paella-player-7/src/js/EpisodeConversor.js

```diff
@@ -278,13 +290,15 @@ function processAttachments(episode, manifest, config) {
       });
     }
     else {
+      // TODO: will this cause an endless recursion loop when there are no
+      // navigation segments preview images within the mediapackage?
       videoPreview = getVideoPreview(episode.mediapackage, config);
     }
```

Comment:
Ref change at https://github.com/opencast/opencast/commit/f4454f61f2f

Reply:
I don't see a problem there, because the function `getVideoPreview` doesn't call any other function recursively.

Reply:
I removed that comment and clarified the names of the variables to avoid confusion about which variable is being set